simphony/cuds/abc_modeling_engine.py

```diff
@@ -22,6 +25,31 @@ class ABCModelingEngine(object):  # pragma: no cover
     """
     __metaclass__ = ABCMeta
 
+    def __init__(self, *args, **kwargs):
```

Comment:
I believe one of the motivations for having `Simulation` to instantiate an modeling engine wrapper internally is because some data from the cuds is required for the instantiation.  This is the part I don't know much about: What are these metadata that are required for instantiation?  Is "use_internal_interface" the only one?  Why can't we ask the user to initialise the modeling engine wrapper themselves?
(The second motivation is for documenting the simulation: what engine is used; in which case `Simulation` could save `repr(modeling_engine)` to somewhere.  And the `__repr__` method of the modeling engine can be overloaded to include more information, which is up to the dev team of the engine)